application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -159,6 +159,14 @@ RestAction<Message> constructChatGptAttempt(ThreadChannel threadChannel,
                     generateDismissButton(componentIdInteractor, idForDismissButton.toString())));
     }
 
+    /**
+     * Generates a MessageEmbed for a response using AI.
+     *
+     * @param answer The response text generated by AI.
+     * @param selfUser The SelfUser representing the bot.
+     * @param title The title for the MessageEmbed.
+     * @return A MessageEmbed that contains response generated by AI.
+     */
```

Comment:
`{@link MessageEmbed}` and `{@link SelfUser}` tags could be used across the JavaDoc for easier clarity and navigation.

Reply:
JDA stuff is best documented on their website, i think we should only document stuff that's custom made.